src/main/java/sysc4806/project24/mini_survey_monkey/models/Response.java

```diff
@@ -0,0 +1,14 @@
+package sysc4806.project24.mini_survey_monkey.models;
+
+import jakarta.persistence.*;
+
+@Entity
+@Inheritance(strategy=InheritanceType.JOINED)
+public abstract class Response {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private int responseId;
```

Comment:
**Could we change this from `responseId` to `id`?**

Argument for change would be to eliminate redundant words.